docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -1784,3 +1784,30 @@ To enable cross origin requests to the remote api add the flag
 "--api-enable-cors" when running docker in daemon mode.
 
     $ docker -d -H="192.168.1.9:2375" --api-enable-cors
+
+
+## 3.4 Errors
+
+If there is an error while processing a request, the daemon will generate
```

Comment:
Another possibility is to just generate a single header, and serialize the Derror as json.  This would allow the same code to produce/consume a Derror without depending on if is in a header, or part of the body; streamed responses will probably need to serialize Derror's anyways.